state-sync/inter-component/consensus-notifications/src/lib.rs

```diff
@@ -321,7 +321,7 @@ mod tests {
 
         // Send a notification
         let notify_result = block_on(consensus_notifier.notify_new_commit(vec![], vec![]));
-        assert!(notify_result.is_ok());
+        notify_result.unwrap();
```

Comment:
If there's some information that would help, expects work great at describing what you wanted.

Reply:
True, the only thing that sucks with `expect` is the need to provide a string message, which a lot of the time, blows up the code size 🤔